trailblazer/services/job_service/job_service.py

```diff
@@ -58,13 +67,26 @@ def _update_slurm_jobs(self, analysis_id: int) -> None:
         jobs: list[Job] = [slurm_info_to_job(job_info) for job_info in slurm_jobs]
         self.store.replace_jobs(analysis_id=analysis_id, jobs=jobs)
 
+    def _update_tower_jobs(self, analysis_id: int) -> None:
+        analysis: Analysis = self.store.get_analysis_with_id(analysis_id)
+        workflow_id: str = get_tower_workflow_id(analysis)
+        tower_jobs: list[TowerJobInfo] = self.tower_service.get_jobs(workflow_id)
```

Comment:
This is kind of janky. Would be nice to encapsulate the logic for retrieving the workflow id in the tower service. Same goes for the slurm job ids.